odoo/addons/test_convert/tests/test_convert.py

```diff
@@ -188,6 +188,36 @@ def test_function_function(self):
         self.assertEqual(args, ())
         self.assertEqual(kwargs, {})
 
+    def test_o2m_sub_records(self):
+        # patch the model's class with a proxy that copies the argument
+        Model = self.registry['test_convert.test_model']
+        call_args = []
+
+        def _load_records(self, data_list, update=False):
+            call_args.append(data_list)
+            return super(type(self), self)._load_records(data_list, update=update)
```

Comment:
This doesn't look like a good idea to me.
All you needed was a comment saying to Pylint "don't worry".

Reply:
```py
# pylint: disable=bad-super-call
```